src/app/lists/page.tsx

```diff
@@ -1,6 +1,6 @@
 import { Metadata } from "next";
 import React, { ReactElement } from "react";
-import ListsDataView from "@/app/lists/ListDataview";
+import ListsDataView, { ListRow } from "@/app/lists/ListDataview";
 import supabase, { Schema } from "@/supabase";
 
 // disables caching
```

Comment:
How come we want no caching at all as opposed to say revalidation every 10 seconds? Not saying it needs to be changed, just a question.

Reply:
I'm not sure as it wasn't me who implemented this

Reply:
@DanWilkins2107 it just means that when we make an edit and reload the page, the edit always appears. We can optimise later if the number of database queries starts to become problematic

Reply:
Ahh makes sense cheers @Var1377